packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1409,14 +1414,17 @@ async function handler(
             }
           }
 
+          const clonedEvent = cloneDeep(evt);
+          const updatedEvent = {
+            ...clonedEvent,
+            additionalNotes, // Resets back to the additionalNote input and not the override value
+            cancellationReason: "$RCH$" + (rescheduleReason ? rescheduleReason : ""), // Removable code prefix to differentiate cancellation from rescheduling for email
+          };
           if (noEmail !== true && (!requiresConfirmation || isOrganizerRescheduling)) {
             const copyEvent = cloneDeep(evt);
-            await sendRescheduledEmails({
-              ...copyEvent,
-              additionalNotes, // Resets back to the additionalNote input and not the override value
-              cancellationReason: "$RCH$" + (rescheduleReason ? rescheduleReason : ""), // Removable code prefix to differentiate cancellation from rescheduling for email
-            });
+            await sendRescheduledEmails(updatedEvent);
           }
+          sendEventScheduledMessages(updatedEvent, getMessagingCredentials(credentials));
```

Comment:
If we use webhooks we won't need to make a change here.

Reply:
These changes were only in `handleSeats` so removing from `handleNewBooking` to fix merge conflict.